prac_09/silver_service_taxi_test.py

```diff
@@ -0,0 +1,15 @@
+"""
+CP1404/CP5632 Practical
+SilverServiceTaxi Test Task
+"""
+from silver_service_taxi import SilverServiceTaxi
+
+
+def main():
+    """test code for SilverServiceTaxi"""
+    my_taxi = SilverServiceTaxi("Prius 1", 100, 2)
+    my_taxi.drive(18)
+    print(f"{my_taxi}, Total Fare Price: ${my_taxi.get_fare()}\n")
```

Comment:
Perhaps test more than one SilverServiceTaxi object, varying the fanciness to make certain it is working properly 😄